patches/node/feat_undici_support_urllist_initialization_in_response_ctor.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Tue, 19 Nov 2024 17:54:33 +0900
+Subject: feat(undici): support urlList initialization in Response ctor
+
+Extends the Response constructor to initialize with urlList array,
+which will be used to support the url getter.
```

Comment:
@codebytere I don't see a way to upstream this change, [Response#init](https://developer.mozilla.org/en-US/docs/Web/API/Response/Response#options) options does not support `urlList`. But we have a unique case of constructing the url list from a different http client (chromium) via undici. Any ideas ?